layered-architectures-module-2/Carting.DAL/Carting.DAL.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
`Layers should be logically separated (via separate folders/namespaces).`

What you did it must be done in Task 2.